pkg/collector/corechecks/snmp/checkconfig/config.go

```diff
@@ -105,9 +115,16 @@ type CheckConfig struct {
 	CollectDeviceMetadata bool
 	DeviceID              string
 	DeviceIDTags          []string
-	Subnet                string
+	ResolvedSubnetName    string
 	AutodetectProfile     bool
 	MinCollectionInterval time.Duration
+
+	Network                  string
+	DiscoveryWorkers         int
+	Workers                  int
+	DiscoveryInterval        int
+	IgnoredIPAddresses       map[string]bool
```

Comment:
I would add `Set` to this variable name, making it explicit it is a dictionary and not an array

Reply:
There is no `Set` in golang, using `map[string]bool` or `map[string]{}` are workaround for this.

I used here the same impl as the one for snmp_listener.

Reply:
I meant in the variable name ;)

Reply:
Ah, ok. I think it's fine to not have type in the name in general (redundant with type). In this case, using `set` in the name might also lead to confusions, since it's not really a set, but a `map[string]bool`.